.travis.yml

```diff
@@ -1,20 +1,42 @@
 language: scala
 
 scala:
-  - 2.11.8
-  - 2.12.1
+- 2.11.8
```

Comment:
I don't think it should be parsed differently. The file was reformatted when I originally ran `travis encrypt-file credentials --add`.